repo/incubating/flink/versions/0/flink-frameworkversion.yaml

```diff
@@ -18,7 +18,38 @@ spec:
     - name: FLINK_JOBMANAGER_REPLICAS
       description: Number of job managers to run
       default: "1"
+    - name: ZOOKEEPER_URL
+      description: Connection information for Zookeeper
+      default: "zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181"
+    - name: ZOOKEEPER_PATH
+      description: Path to store Flink data in Zookeeper
+      default: "/flink"
+    - name: HIGH_AVAILABILITY
+      description: Defines high-availability mode used for the cluster execution. To enable high-availability, set this mode to "ZOOKEEPER" or specify FQN of factory class.
+      default: NONE
```

Comment:
Any particular difference between `NONE` and `"NONE"` ? I am asking because all other default values are set as strings within `""`